requirements/run.txt

```diff
@@ -35,6 +41,8 @@ flask-cors==3.0.10
     # via -r requirements/run.in
 flask-socketio==5.2.0
     # via -r requirements/run.in
+idna==3.4
```

Comment:
Let's pin `idna==3.3` for now until a next NApp run dependencies upgrade, otherwise it could break the pip resolution on Dockerbuild:

```
❯ rg --follow "idna" -g "*.txt" | rg "requirements/run.txt"
napps/amlight/sdntrace_cp/requirements/run.txt:idna==3.3
napps/kytos/maintenance/requirements/run.txt:idna==3.3
napps/kytos/mef_eline/requirements/run.txt:idna==3.3
napps/kytos/of_lldp/requirements/run.txt:idna==3.3                 # via requests
napps/amlight/sdntrace/requirements/run.txt:idna==3.3
napps/amlight/coloring/requirements/run.txt:idna==3.3
```

Once you pin and adapt the other transitive dependencies, I'll encourage you to [`docker build`](https://github.com/amlight/kytos-docker/blob/master/Dockerfile) with your branch to make sure it's building successfully with other NApps, this is great way to confirm that we've selected and pinned compatible and transitive dependencies.

Reply:
Understood. I will run docker.

Reply:
@Alopalao did you have a chance to docker build and confirm that the current dependencies are all set? Waiting for this confirmation before we close this thread.

Reply:
Sorry, I totally forgot. Here it is, from 18/10/2022:
```
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /tests
plugins: timeout-2.0.2
collected 200 items

tests/test_e2e_01_kytos_startup.py ..                                    [  1%]
tests/test_e2e_05_topology.py ..................                         [ 10%]
tests/test_e2e_10_mef_eline.py .........Xss.....x....x.E...              [ 24%]
tests/test_e2e_11_mef_eline.py .....                                     [ 26%]
tests/test_e2e_12_mef_eline.py .....xx.                                  [ 30%]
tests/test_e2e_13_mef_eline.py .....xxXx......xxXx.XXxX.xxxx..x......... [ 51%]
...                                                                      [ 52%]
tests/test_e2e_14_mef_eline.py x                                         [ 53%]
tests/test_e2e_15_maintenance.py ........................                [ 65%]
tests/test_e2e_15_mef_eline.py .                                         [ 65%]
tests/test_e2e_20_flow_manager.py ................                       [ 73%]
tests/test_e2e_21_flow_manager.py ..                                     [ 74%]
tests/test_e2e_22_flow_manager.py ...............                        [ 82%]
tests/test_e2e_23_flow_manager.py X......................                [ 93%]
tests/test_e2e_30_of_lldp.py ....                                        [ 95%]
tests/test_e2e_31_of_lldp.py ...                                         [ 97%]
tests/test_e2e_32_of_lldp.py ...                                         [ 98%]
tests/test_e2e_40_sdntrace.py X.x                                        [100%]
```